packages/core/lib/generators/integrations/flare_integration.dart

```diff
@@ -65,10 +65,11 @@ ${isPackage ? "\n  static const String package = '$packageParameterLiteral';" :
   String get className => 'FlareGenImage';
 
   @override
-  String classInstantiate(String path) => 'FlareGenImage(\'$path\')';
+  String classInstantiate(AssetType asset) =>
+      'FlareGenImage(\'${asset.posixStylePath}\')';
 
   @override
-  bool isSupport(AssetType type) => type.extension == '.flr';
+  bool isSupport(AssetType asset) => asset.extension == '.flr';
```

Comment:
Is this intentional?

Reply:
Ah, yes, I think I used the word 'asset' in a few new places, as it represents a asset, not a type of asser. But happy to change back.